docs/testing.rst

```diff
@@ -0,0 +1,97 @@
+=======
+Testing best practices
+=======
+
+Test set up
+===========
+
+Doing a lot of work in the ``setUp`` call of a test class means that it will be run on every test. This
+quickly adds a lot of run time to the tests. Some things that can be easily moved to ``setUpClass`` are domain
+creation, user creation, or any other static models needed for the test.
+
+Sometimes classes share the same base class and inherit the ``setUpClass`` function. Below is an example:
+
+.. code:: python
+
+    # BAD EXAMPLE
+
+    class MyBaseTestClass(TestCase):
+
+        @classmethod
+        def setUpClass(cls):
+            ...
+
+        def _my_helper_function(self):
+            ...
+
+
+    class MyTestClass(MyBaseTestClass):
+
+        def test(self):
+            ...
+
+    class MyTestClassTwo(MyBaseTestClass):
+
+        def test(self):
+            ...
+
+
+In the above example the ``setUpClass`` is run twice, once for ``MyTestClass`` and once for ``MyTestClassTwo``. If ``setUpClass`` has expensive operations, then it's best for all the tests to be combined under one test class.
```

Comment:
I hate to keep beating this horse, but the example below does not combine all tests "under one test class" as this sentence describes. It has two test classes with separate `setUpClass` methods.

This code example appears to be demonstrating how to share a helper method with multiple test classes using a mixin, which is a good technique to document. However, it seems like a separate point from merging tests to avoid running expensive setup repeatedly.